### PR TITLE
LPS-175992 All fields are clean on entries when Formula Field is wrong

### PR DESCRIPTION
**Pull request by davisaints**

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:object

**Comment by davisaints** [comment]
ci:test:relevant

**Comment by davisaints** [comment]
ci:test:object

**Comment by davisaints** [comment]
ci:test:relevant

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 1]
```diff
@@ -1046,15 +1070,35 @@ public void testAddObjectEntryWithFormulaObjectField() throws Exception {
...
+			).put(
+				"listTypeEntryKeyRequired", "listTypeEntryKey1"
+			).build());
+
 		Assert.assertEquals(
 			75D,
 			MapUtil.getDouble(
 				_objectEntryLocalService.getValues(
-					objectEntry.getObjectEntryId()),
+					objectEntry1.getObjectEntryId()),
```
Comment (pedro-oliveira446) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:1088]: Alphabetical order here as well.

**Review by pedro-oliveira446 — CHANGES_REQUESTED** [review]
Great job @davisaints, only small changes.

File: modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java [hunk 2]
```diff
@@ -1035,7 +1035,31 @@ public void testAddObjectEntryWithFormulaObjectField() throws Exception {
 					).build())
 			).build());
 
-		ObjectEntry objectEntry = _addObjectEntry(
+		ObjectField objectField2 = _addCustomObjectField(
+			new FormulaObjectFieldBuilder(
+			).labelMap(
+				LocalizedMapUtil.getLocalizedMap("Able")
+			).name(
```
Comment (pedro-oliveira446) [inline modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java:1042]: I'm not sure about the field name, I believe that something like `id sum` is better.

**Review by pedro-oliveira446 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**